src/UI/Component/Chart/Factory.php

```diff
@@ -28,4 +28,32 @@
 	 * @return  \ILIAS\UI\Component\Chart\ScaleBar
 	 */
 	public function scaleBar(array $items);
+
+
+    /**
+     * ---
+     * description:
+     *   purpose: >
+     *     Gauges are used to display a progress or performance.
```

Comment:
We should not choose a name for such an element that is already used by a specific feature in ILIAS. Otherwise we could also call it Learning Objective Indicator - because this is where it will appear first. In my understanding, we already have learning progress indicators - our so-called "Pöppel" in grey, yellow, green and red. These are learning progress indicators. 
Chris' suggestion to use something with "...meter" convinces me - because this is how it looks like. And it has no relation to an existing feature - same like all the other UI components we already have.

Reply:
The Pöppel is a Learning Progress Status Display. The Pöppel is not measuring progress but indicates a status.

Reply:
If something indicates something, it is an indicator. Measuring is done by none of our UI elements but of the programme itself. The general difference between the pöppel and the meter is that the first can only display one status (presented by colour) while the second allows to present a value on a scale. This is why such an element is good for all cases where percentages or points should be shown and where an additional status related on the reached percentage/points should be presented (bar turns to green if required level is reached for example). Score Meter or Progress Meter would be fine for me, too.

Reply:
The Progress Bar for the download is also a Progress Meter with a max value and a current value. We have currently just this element. There are several obvious candidates: To allow distinguishing between the Speedo, the Pöppel, and the Download Progress bar the names must not be too generic. There will be more Progress Indicators at some point in time.